packages/fxa-payments-server/src/routes/Product/index.tsx

```diff
@@ -59,6 +59,8 @@ export const Product = ({
     AppContext
   );
 
+  const [transactionInProgress, setTransactionInProgress] = useState(false);
```

Comment:
Can this be in `SubscriptionCreate`?  I don't see why it should be in here.

Reply:
I had it here so that it can be unset by `customerIsSubscribedToProduct` [here](https://github.com/mozilla/fxa/pull/7613/files/cd99a322f208e000d4dc68211dd9748be93530cb#diff-9cb71f54629ac6b7b6ccf9d0d5e3d30ed33d99367527bb55519116778955e5d2R82). I guess we don't really need to unset, since it will be replaced by `SubscriptionSuccess`

Reply:
`customerIsSubscribedToProduct` is just a lib function.  It doesn't use anything from the scope of `Product`.  We can move it somewhere so that both `Product` and `SubscriptionCreate` can use it, if that's the issue.